tests/suites/1_stateful/06_read_parquet/06_0000_read_parquet.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+aws --endpoint-url http://127.0.0.1:9900/ s3 cp s3://testbucket/admin/data/ontime_200.parquet /tmp/ontime_200.parquet  > /dev/null 2>&1
+
+echo "select count(*) from read_parquet('/tmp/ontime_200.parquet')" | $MYSQL_CLIENT_CONNECT
```

Comment:
It's better rename as `06_0000_read_parquet_ontime_200.parquet` to avoid conflict between different tests.